app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/item/DateSessionsSection.kt

```diff
@@ -10,18 +10,18 @@ class DateSessionsSection : Section() {
     fun updateSessions(
             sessions: List<Session>,
             onFavoriteClickListener: (Session.SpeechSession) -> Unit,
-            simplify: Boolean = false
+            isShowDayNumber: Boolean = false
     ) {
         val sessionItems = sessions.map {
             when (it) {
                 is Session.SpeechSession -> {
                     @Suppress("USELESS_CAST")
                     SpeechSessionItem(
```

Comment:
Cloud you use named argument?

Reply:
@takahirom 
thank you review comment. but, your comment is unclear for me.. 😥
isShowDayNumber is already "named argument".
If the intention is different, will you present a code example?

I changed the name of updateSession()'s argument "simplify" to "isShowDayNumber".
Because "simplify" is used for a different purpose(?) in the constructor of SpeechSessionItem.

Reply:
Sorry 🙇 
I want to do like this

```
SpeechSessionItem(
                            session = it, 
                            onFavoriteClickListener = onFavoriteClickListener, 
                            isShowDayNumber = simplify
                    ) as SessionItem
```